PROJECT_LIFECYCLE.md

```diff
@@ -155,6 +155,31 @@ You should use the [![Archived badge](https://img.shields.io/badge/Stage-Archive
 [![Archived badge](https://img.shields.io/badge/Stage-Archived-orange)](https://github.com/eiffel-community/community/blob/master/PROJECT_LIFECYCLE.md#stage-archived)
 ```
 
+#### Process for archiving a repository
+
+- Create ticket in the repository to be archived. See [This repo should be archived eiffel-store#10](https://github.com/eiffel-community/eiffel-store/issues/10) for an example
+- Send mail to maintainers google group, with the following contents (as decided in TC on [Feb 27](https://github.com/eiffel-community/community/blob/master/meetings/MEETINGS_TC_2024.md#action-items-7)):
+
+> (Subject: Archiving of Eiffel Store)
+> Hi maintainers!
+> 
+> We would like to make you aware we have created a ticket in your repository [eiffel-community/eiffel-store#10](https://github.com/eiffel-community/eiffel-store/issues/10) on archiving it. Please use this ticket for further communication on this topic.
+> 
+> Best Regards,  
+> Eiffel Technical Committee
+
+- Create a PR that updates the Readme with the following changes:
+    - Change the badge to the archived badge 
+- Close all issues
+- Close all PRs
+- Follow the instructions on https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories
+- Open a PR to in the [eiffel-community/eiffel-community.github.io](https://github.com/eiffel-community/eiffel-community.github.io) repository to remove it from [https://eiffel-community.github.io/community.html](https://eiffel-community.github.io/community.html)
+- Delete the google group: Leaving the google group can cause confusion on if the repository has maintainers or not. Removing the group makes this clear.
```

Comment:
Here and in a one or two other places we use the all-lowercase "google group" but below it's "Google Group". I think we should stick with the latter, but "Google group" would also be okay.